src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -208,7 +209,10 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     eventually {
       val createdCluster = leo.getActiveClusterDetails(userInfo, project, name1).futureValue
 
-      createdCluster.clusterImages.map(_.dockerImage) should contain (dataprocConfig.welderDockerImage)
+      // cluster images should contain welder and Jupyter
```

Comment:
Updated tests to verify presence of the Jupyter image. I verified that these tests _failed_ without the fix in `LeonardoService`.